src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -934,8 +934,22 @@ void program_node::init_onednn_primitive_attributes() {
                 post_ops.append_eltwise(dnnl::algorithm::eltwise_clip, 0.0f, 1.0f);
                 update_onednn_post_op_list(onednn_post_op_type::eltwise_linear, empty_mem);
                 update_onednn_post_op_list(onednn_post_op_type::eltwise_clip, empty_mem);
+            } else if (fused_desc->activation_function == cldnn::activation_func::hsigmoid) {
```

Comment:
I think we can use onednn `hard_sigmoid` for this case. Please see `eltwise_hardswish` example below.